lib/builder/items.rb

```diff
@@ -18,13 +18,13 @@ def build
       item
     end
 
-    def add_item(item)
-      @item_details = item
+    def add_item(item_details:)
```

Comment:
My meaning here probably got lost in translation - as in named parameters for each part of the "item details"